lib/node_modules/@stdlib/blas/ext/base/dapxsumkbn2/manifest.json

```
@@ -1,40 +1,45 @@
 {
-	"options": {},
-	"fields": [
-		{
-			"field": "src",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "include",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "libraries",
-			"resolve": false,
-			"relative": false
-		},
-		{
-			"field": "libpath",
-			"resolve": true,
-			"relative": false
-		}
-	],
-	"confs": [
-		{
-			"src": [
-				"./src/dapxsumkbn2.c"
-			],
-			"include": [
-				"./include"
-			],
-			"libraries": [
-				"-lm"
-			],
-			"libpath": [],
-			"dependencies": []
-		}
-	]
+    "options": {},
+    "fields": [
+        {
+            "field": "src",
+            "resolve": true,
+            "relative": true
+        },
+        {
+            "field": "include",
+            "resolve": true,
+            "relative": true
+        },
+        {
+            "field": "libraries",
+            "resolve": false,
+            "relative": false
+        },
+        {
+            "field": "libpath",
+            "resolve": true,
+            "relative": false
+        }
+    ],
+    "confs": [
+        {
+            "src": [
+                "./src/dapxsumkbn2.c"
+            ],
+            "include": [
+                "./include"
+            ],
+            "libraries": [
+                "-lm"
+            ],
+            "libpath": [],
+            "dependencies": [
+                "stdlib/napi/export.h",
+                "stdlib/napi/argv.h",
+                "stdlib/napi/argv_int64.h",
+                "stdlib/napi/argv_strided_float64array.h"
```

The dependencies should be listed as stdlib packages:

```suggestion
                "@stdlib/napi/export",
                "@stdlib/napi/argv",
                "@stdlib/napi/argv-int64",
                "@stdlib/napi/argv-strided-float64array"
```